python/cudf/cudf/tests/test_string.py

```diff
@@ -847,10 +793,6 @@ def test_string_upper(ps_gs):
 @pytest.mark.parametrize("n", [-1, 0, 1, 3, 10])
 @pytest.mark.parametrize("expand", [True, False, None])
 def test_string_split(data, pat, n, expand):
-
-    if data in (["a b", " c ", "   d", "e   ", "f"],) and pat is None:
-        pytest.xfail("None pattern split algorithm not implemented yet")
```

Comment:
version?

Reply:
I think this was a cudf related xfail and when `pat` is `None` support was added in cudf we might have missed to remove this xfail. So don't think we need a version here.